### PR TITLE
Add .py extension to hacking/test-module

### PR DESCRIPTION
**Pull request by prasadkatti**
##### SUMMARY
Add .py file extension to hacking/test-module

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME
`hacking/test-module`

##### ANSIBLE VERSION

```
ansible 2.5.0
```

### CODE REVIEW

**Comment by daxlab** [comment]
The file already has a python hashbang.

File: hacking/test-module.py [hunk 1]
```diff
@@ -58,21 +58,22 @@ def parse():
     parser.usage = "%prog -[options] (-h for help)"
 
     parser.add_option('-m', '--module-path', dest='module_path',
-        help="REQUIRED: full path of module source to execute")
+                      help="REQUIRED: full path of module source to execute")
```
Comment (daxlab) [inline hacking/test-module.py:61]: The file already has a python hashbang.

**Comment by sivel** [comment]
I'm not sure why this was done.  This PR seems to be lacking a reasoning for doing this.  Additionally, many people expect this file to not have an extension, so if for whatever reason there is to require a .py file extension, this file should likely be symlinked.

needs_info

**Comment by prasadkatti** [comment]
Hi @sivel - Yes. There is no particular reason for this PR. I saw that all the other python files in `hacking` have the .py extension and therefore thought this one should have too. Also, isn't it a fairly standard convention to have a .py extension to python files? Also I was browsing through the repo and saw this commit 213e5181654dfb4ed97c7ed7cbc71e9a0edd612f

You are right. The file should have been symlinked as part of this change, because people expect the file to be w/o extension. 

Feel free to close the PR if this seems unnecessary. Sorry about the trouble.

**Comment by prasadkatti** [comment]
Also, its not a bugfix at all (since there is no bug), but none of the other 3 seem to be apply to this either. :(

**Comment by sivel** [comment]
Thank you very much for the submission, however we are going to pass on this for now.

If you have further questions please stop by IRC or the mailing list:

   * IRC: #ansible on irc.freenode.net
   * mailing list: https://groups.google.com/forum/#!forum/ansible-project

**Pull request closed without merging**